patternlab/styleguide/source/_patterns/03-organisms/by-author/related-orgs-and-topics.twig

```diff
@@ -0,0 +1,18 @@
+<section class="ma__related-orgs-and-topics">
+  <h2 class="ma__comp-heading">{{ relatedOrgsAndTopics.header }}</h2>
```

Comment:
Is there any reason not to use `01-atoms/04-headings/comp-heading.twig` instead of hard-coding here?